src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Esfa.Recruit.Provider.Web.ViewModels.Reports.ProviderApplicationsReport;
+using Esfa.Recruit.Vacancies.Client.Infrastructure.Client;
+
+namespace Esfa.Recruit.Provider.Web.Orchestrators.Reports
+{
+    public class ProviderApplicationsReportOrchestrator
+    {
+        private readonly IRecruitVacancyClient _vacancyClient;
+
```

Comment:
Do we need an instance of _vacancyClient when it is not being used?

Reply:
It will be. I'm just getting the scaffolding in place